ui/desktop/electron-app/src/index.js

```diff
@@ -92,8 +92,13 @@ const createWindow = (partition, closeWindowCB) => {
 
   const browserWindow = new BrowserWindow(browserWindowOptions);
 
+  // TODO: Figure out how to spawn more than one terminal process in a browser window
+  //       while keeping track of handlers
   // Terminal
-  const shell2 = os.platform() === 'win32' ? 'powershell.exe' : 'bash';
+  const shell2 =
+    os.platform() === 'win32'
+      ? 'powershell.exe'
+      : process.env.SHELL || '/bin/bash';
```

Comment:
Should we just use `bash` or use the user's shell? If they have something like a custom `PS1` it might not play nicely.

Reply:
I also agree with using bash, at least for now.